src/utils/formatting-doc-to-txt.ts

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+
+const MAX_ANSWERS = 8;
+
+const inputFilename = 'input.txt';
+const outputFilename = 'output.csv';
+
+// Read the input file and split it into lines
+const lines = fs.readFileSync(inputFilename, 'utf8').split('\n');
+
+// Create an array to hold the data for the output CSV file
+var outputData = [];
+
+// Initialize the line index
+var i = 0;
+
+while (i < lines.length) {
+    // Check if this is a question line
+    if (lines[i].includes('.')) {
+        var question = lines[i].trim().replace(',', '');
+
+        console.log(`Question detected: ${question}`);
+
+        var answers = [];
+        var points = [];
+
+        // Loop through the next 5 lines to get the answers and points
+        for (var j = i + 1; j < i + MAX_ANSWERS; j++) {
+            try {
+                var line = lines[j].trim().replace('—', '').replace('â€”', '').replace('""', '').replace(',', '').replace("   ", '');
+                var [answer, point] = line.split(' ');
+
+                console.log(`Answer: ${answer} Point: ${point}`);
+
+                answers.push(answer);
+                points.push(parseInt(point)); // Parse the point as a number
+            } catch (error) {
+                break;
+            }
+        }
+
+        // Sort answers by points in descending order
+        var sortedAnswers = [];
```

Comment:
Can be `const sortedAnswers`, in Javascript const arrays do not mean they are imutable, but the pointer in memory to that array will always be the same. Read more [here](https://www.tutorjoes.in/JS_tutorial/use_of_const_for_creating_arrays_in_javascript)